x/blob/README.md

```diff
@@ -191,7 +191,7 @@ The blob module emits the following events:
 |---------------|-----------------------------------------------|
 | signer        | {bech32 encoded signer address}               |
 | blob_sizes    | {sizes of blobs in bytes}                     |
-| namespace_ids | {namespaces the blobs should be published to} |
+| namespaces    | {namespaces the blobs should be published to} |
```

Comment:
thanks for this @sayeedkhannabil while you're at it, do mind also updating the name on line 188? it should be EventPayForBlobs

Reply:
Sure. I am on it.